src/__pisa_tests__/quickTest.ts

```diff
@@ -0,0 +1,90 @@
+import * as chai from "chai";
+import "mocha";
+import request from "request-promise";
+import { KitsuneTools } from "../kitsuneTools";
+import { ethers } from "ethers";
+import { IConfig } from "../dataEntities/config";
+const config = require("./../../config.json") as IConfig;
+const provider = new ethers.providers.JsonRpcProvider(config.jsonRpcUrl);
```

Comment:
please use the providers file to get the provider

Reply:
agreed, added